### ISSUE TITLE
ngx-graph displays correctly only after resizing Chrome browser window

### ISSUE THREAD

**Issue by alienninja** [swimlane/ngx-graph#85]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
Chrome 66.0.3359.170:
Upon first opening the web browser to view the graph, all the lines and node labels are present, however the color boxes for each node are not present.  Resizing the browser window forces a refresh and then the graph is displayed correctly

**Expected behavior**
Without this issue, first opening the browser, the graph should display correctly.

**Reproduction of the problem**
100% repeatable

**Please tell us about your environment:**
Windows 10,
Angular CLI: 1.6.0
Node: 8.11.1
OS: win32 x64
Angular: 5.2.10
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cdk: 5.2.5
@angular/cli: 1.6.0
@angular/material: 5.2.5
@angular-devkit/build-optimizer: 0.0.42
@angular-devkit/core: 0.0.29
@angular-devkit/schematics: 0.0.52
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.9.0
@schematics/angular: 0.1.17
@swimlane/ngx-graph: 4.3.0
typescript: 2.4.2
webpack: 3.10.0

Repeatable on Chrome 66.0.3359.170
Firefox 60.0, works correctly

**Comment by rinkako** [comment]
I've encountered this problem also, and I solved it finally. I hope it is helpful to you.
ngx graph will not draw node correctly before resizing the page when it is put into some container. In my case, I put the `ngx-graph` into a `tab`(ngx-bootstrap/tabs) container, and the problem in this issue appears. After I move `ngx-graph` outside `tab`, it acts correctly.

**Comment by teslovich** [comment]
or add css for chart's container
```
position: absolute;
top: 50%;
left: 50%;
transform: translate(-50%, -50%)
```

**Comment by assafsun** [comment]
Closing this issue as it seems like a solution was offered.
Please create a new issue or update this one if something was missed.

**Comment by animbalk** [comment]
We are facing this display issue on chrome. We can not add absolute css property in container or add/remove parent containers as it disturb our layout. Please suggest a solution.
